s3update.go

```diff
@@ -160,25 +159,8 @@ func runAutoUpdate(u Updater) error {
 		fmt.Printf("s3update: updated with success to version %d\nRestarting application\n", remoteVersion)
 
 		// The update completed, we can now restart the application without requiring any user action.
-		var args []string
-		if len(os.Args) > 1 {
-			args = os.Args[1:]
-		}
-		cmd := exec.Command(os.Args[0], args...)
-		cmd.Stdin = os.Stdin
-		cmd.Stderr = os.Stderr
-		cmd.Stdout = os.Stdout
-		if err := cmd.Run(); err != nil {
-			// If the command fails to run or doesn't complete successfully, the
-			// error is of type *ExitError. Other error types may be
-			// returned for I/O problems.
-			if exiterr, ok := err.(*exec.ExitError); ok {
-				// The command didn't complete correctly.
-				// Exiting while keeping the status code.
-				os.Exit(exiterr.Sys().(syscall.WaitStatus).ExitStatus())
-			} else {
-				return err
-			}
+		if err := syscall.Exec(os.Args[0], os.Args, os.Environ()); err != nil {
```

Comment:
Shouldn't the second arg be `os.Args[1:]`?
I never used it so, not sure

Reply:
I do not know why but the second argument as to be the whole `os.Args` array, not the one without the exec.

I supposed it's necessary to properly replace the process 🤷‍♂ 

Tried locally both version, the one using `os.Args[1:]` was failing.

According to both documentation and method signature it seems consistent at least : 
* https://golang.org/src/syscall/exec_unix.go?s=7396:7461#L252
* https://golang.org/src/os/exec/exec.go?s=6250:6295#L158

Reply:
Ok, good to know! Thanks

Reply:
Is it ok for you ? You did not approved the review yet 😄 🤔

Reply:
oops sorry :D

Reply:
No pb 😄 I was just wondering if something else catches your eyes